CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Changed
+
+- Show correct placeholder in legacy consumer in input field template. (#5924)
+- Do not allow choosing placeholder in legacy consumer in select field template. (#5924)
```

Comment:
If we end up implementing my suggestions and comments below, please update these to reflect the actual change (if we revert the changes, then remove the changelog item).

Reply:
updated: https://github.com/impress-org/givewp/compare/62ee48bf965c...26e1246afb97